.gitignore

```diff
@@ -6,6 +6,7 @@ jetpack_vendor/
 /.nova/
 /logs
 /allure-results/
+/results
```

Comment:
Let's name this directory something a little more specific. If it has to go inside the monorepo root at all, instead of generating the report somewhere outside the monorepo.